netbench-cdk/lib/netbench.ts

```diff
@@ -73,8 +85,16 @@ export class NetbenchInfra extends cdk.Stack {
             ],
 
         });
-        //Netbench Orchistrator is expecting only one tagged subnet.
-        cdk.Tags.of(vpc.publicSubnets[0]).add('aws-cdk:subnet-name', 'public-subnet-for-runners');
+        
+        //Tag all available subnets the same. This behavior might need to change when MultiRegion is added.
+        const subnetTagKey = "aws-cdk:netbench-subnet-name";
+        const subnetTagValue = "public-subnet-for-netbench-runners";
+        vpc.publicSubnets.forEach(element => {
+          cdk.Tags.of(element).add(subnetTagKey, subnetTagValue);
+        });
+        new cdk.CfnOutput(this, "output:NetbenchSubnetTagKey", { value: subnetTagKey});
+        new cdk.CfnOutput(this, "output:NetbenchSubnetTagValue", { value: subnetTagValue});
+        new cdk.CfnOutput(this, "output:"+this.stackName+"Region", { value: this.region});
```

Comment:
Each stack is region specific, this allows us to add more stacks in other regions and get the output.